mycroft/skills/intent_service.py

```diff
@@ -409,7 +409,7 @@ def handle_add_context(self, message):
         entity['match'] = word
         entity['key'] = word
         entity['origin'] = origin
-        self.adapt_service.context_manager.inject_context(entity)
+        self.adapt_service.add_context(entity)
```

Comment:
method does not seem to exist in AdaptService

Reply:
Thanks, this was in the first draft but seem to have missed updating here. will fix